src/huggingface_hub/hf_api.py

```diff
@@ -1180,6 +1181,18 @@ def model_info(
 
         Returns:
             [`huggingface_hub.hf_api.ModelInfo`]: The model repository information.
+
+        <Tip>
```

Comment:
should this be a tip or in the `Raises` section? (same for the following changes)

Reply:
So far we have been using `<Tip>` across the codebase as `Raises` wasn't working with doc-builder. It's my fault, it seems I was delaying https://github.com/huggingface/doc-builder/pull/141.

If it's fine with you, I'll merge it like this and open an issue to modify all of those `<Tip>` to `Raises:` once the PR above is merged.